src/ocean/sys/GetIfAddrs.d

```diff
@@ -28,7 +28,44 @@ import core.stdc.string;
 import ocean.stdc.posix.arpa.inet;
 import ocean.stdc.posix.netinet.in_: sockaddr_in, sockaddr_in6;
 import ocean.stdc.posix.sys.socket;
-import core.sys.linux.ifaddrs;
+
+static if (__VERSION__ >= 2000 && __VERSION__ < 2073)
```

Comment:
Ha, yes, shame we don't have a way to officially backport at least the bindings in the previous version - they are too tightly coupled with compiler releases IMHO. Perhaps TODO here? "TODO: remove this when drop support for 2073" or similar.